Project21/Project21/intlist.h

```diff
@@ -0,0 +1,34 @@
+// ------------------------------------------------------------------------------------------------
+// Assignment 2 ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// intlist.h
+// c file for creating a linked list
+// ------------------------------------------------------------------------------------------------
+#pragma once
+#include <stdbool.h>
+
+#define E_INDEX_OUT_OF_RANGE  -1
+#define E_EMPTY_LIST -2
+#define E_LIST_DOES_NOT_EXIST -3
+#define E_VALUE_NOT_FOUND -4
+
+typedef struct Node {

```

Comment:
The documentation comments should be in the header file so the user is able to access the function descriptions while using them. Please go through the links on documentation I provided on the group